go.mod

```diff
@@ -0,0 +1,60 @@
+module github.com/docker/swarmkit
```

Comment:
actually one thing I'm just thinking of; if we want to move this repository to the moby org; would it be better to move and change the module name at the same time (so move first, then add module?)

@AkihiroSuda you recently renamed some repos; any recommendation?

Reply:
Yes, I suggest renaming module name as well